pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -179,16 +182,14 @@ struct AlwaysTrue : Matcher {
   bool match(Operation *op) const final { return true; }
 };
 
-template <typename InnerPredicate>
-struct ResultIs : Matcher {
+template <typename InnerPredicate> struct ResultIs : Matcher {
```

Comment:
I'm wondering why these format changes were made? Did you use `clang-format`?

Reply:
Yes. That's the result of my clang-format.

Reply:
We use `clang-format-9`, which version were you using?

Reply:
It's clang-format version 6.0.0. I'll update my version.